frontend/src/EthereumContext.tsx

```diff
@@ -1,12 +1,20 @@
 import { createContext, useContext, useState } from "react";
 import Web3 from "web3";
-import { setupContracts } from "./utils/web3-helpers";
+import {
+  DynamicContracts,
+  createContract,
+  setupContracts,
+} from "./utils/web3-helpers";
+import { Event } from "../src/screens/AdminEventsPanel";
 
 interface EthActions {
   walletAddress?: string;
   connectWallet: () => Promise<void>;
   createRanks: (params: CreateRanksRequest) => Promise<void>;
+  createTestTicketContract: (params: any) => Promise<void>;
```

Comment:
Please refrain from using `any` type. Prefer generic or `unknown`.

Reply:
I'll be avoiding this in the future, but this I'll remove this request after merging https://github.com/war-in/suu-nft-project/pull/11 , because I needed that function only to mock ticket object on contract's side.